examples/cli.py

```diff
@@ -39,7 +40,7 @@ def main():
     password = arguments['<password>']
     RPC = arguments['<rpc>']
 
-    client = CiscoGRPCClient(IP, TCP_PORT, 10, user, password)
+    client = CiscoGRPCClient(IP, TCP_PORT, 600, user, password)
```

Comment:
You should think about add timeout as an ARG value and having a default of 10 or 600 or whatever that way you can override it if needed.